src/components/tools/Charts/LineArea/Chart.svelte

```diff
@@ -34,6 +36,8 @@
     tickFormat: timeFormat("%Y"),
     units: "",
   };
+  //export let annotations;
```

Comment:
Did you mean to make `annotations` an exportable property? Otherwise this would mean every LineArea chart would have sample annotations applied, correct?

Reply:
`annotations` would be an exportable property. The `getSampleAnnotations` is just for testing.

Reply:
Thanks for explaining.